proto/ignition/msgs/wheel_slip_parameters_cmd.proto

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "WheelSlipParametersCmd";
+
+/// \ingroup ignition.msgs
+/// \interface WheelSlipParametersCmd
+/// \brief Message containing a wheel slip parameters user command.
+
+import "ignition/msgs/header.proto";
+
+message WheelSlipParametersCmd
+{
+  /// \brief Optional header data
+  Header header                       = 1;
+
+  string model_name                   = 2;
+  string link_name                    = 3;
+
+  double slip_compliance_lateral      = 4;
+  double slip_compliance_longitudinal = 5;
```

Comment:
Sorry for the late comment, but while reviewing the forward-port, I noticed that most fields on this message aren't documented.

Reply:
I can add docs in a new PR

Reply:
See https://github.com/ignitionrobotics/ign-msgs/pull/227